src/Http/Role/EditRoleRestController.php

```diff
@@ -115,13 +115,13 @@ public function addPermission(string $roleId, string $permissionKey): Response
             throw new InvalidArgumentException('Required field roleId is missing');
         }
 
-        if (!in_array($permissionKey, array_keys(Permission::getConstants()))) {
+        if (!array_key_exists($permissionKey, Permission::getAllowedValues())) {
```

Comment:
I think this array `Permission::getAllowedValues()` is not associative and has integer values as keys. So `array_key_exists` maybe always returns `true`?